schemas/test/en/test_supplementary_data.json

```diff
@@ -58,11 +58,52 @@
                                 "contents": [
                                     {
                                         "title": "You have successfully loaded Supplementary data",
-                                        "description": "Press continue to proceed to the introduction",
+                                        "description": {
+                                            "text": "List of products: {products}",
```

Comment:
With other list (employees) we run into the problem of first names and then last names (in case of employees) being rendered in sequence, so, for those not involved in this conversation before, list like: [`first name person a`, `first name person b`, `last name person a`, `last name person b`], as it first loops through the first then the second, that's how it was designed. As long as we are happy, for example, with only last names on the list, then I can add it, otherwise some bigger changes will be needed.

Reply:
Yes, that's fine.